apps/angular/src/app/features/anime/anime.module.ts

```diff
@@ -4,14 +4,33 @@ import { AnimeService } from '@js-camp/angular/core/services/anime-service';
 import { HttpClientModule } from '@angular/common/http';
 import { CommonModule } from '@angular/common';
 import { MatTableModule } from '@angular/material/table';
+import { MatPaginatorModule } from '@angular/material/paginator';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MatSortModule } from '@angular/material/sort';
+import { MatSelectModule } from '@angular/material/select';
+import { MatFormFieldModule, MAT_FORM_FIELD_DEFAULT_OPTIONS } from '@angular/material/form-field';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatInputModule } from '@angular/material/input';
 
 import { AnimeDashboardComponent } from './anime-dashboard/anime-dashboard.component';
 
 /** Anime table module. */
 @NgModule({
-	imports: [HttpClientModule, CommonModule, MatTableModule],
+	imports: [
+		HttpClientModule,
+		CommonModule,
+		MatTableModule,
+		MatPaginatorModule,
+		BrowserAnimationsModule,
+		MatSortModule,
+		MatSelectModule,
+		MatFormFieldModule,
+		FormsModule,
+		ReactiveFormsModule,
```

Comment:
As far as I can see you don't use template driven forms in your app.
So we can remove 'FormsModule' import as it's redundant for reactive forms.
```suggestion
		ReactiveFormsModule,
```

You can read this [article](https://blog.angular-university.io/introduction-to-angular-2-forms-template-driven-vs-model-driven/) to get more details about their differences.